sonarts-core/tests/rules/noDuplicateInCompositeRule/noDuplicateInCompositeRule.lint.ts

```diff
@@ -16,6 +7,64 @@ interface Loggable<T> {
   log(param: T): string;
 }
 
+type nokUDuplicate = number | number | string;
+// [10:30-10:36] {{Remove this duplicated type or replace with another one.}}
```

Comment:
@andrea-guarino-sonarsource replace all primary issue locations with "^^^" syntax and keep secondary locations only in 2-3 places